### DISCUSSION TITLE
BCs in tabular format with PorousFlowSink

### DISCUSSION THREAD

**Discussion by vabufano** [idaholab/moose#26076] [Q&A Modules: Porous Flow] [answered]
Hi all,
as part of a seepage analysis I am performing, I need to apply time-varying flows (constant within each time interval) to an edge of my model. 
So far, the solution I have found is to set different boundary conditions, each for the various flow values I want to apply, and then enable and disable the conditions with the Control System. However, I would like to set this boundary condition in tabular format, associating each flow value with a time duration, without using the Control System.
Would it be possible to do something like this?

**Comment by cpgr** [comment, marked as answer]
Have you tried a Function(you can even read a csv file if necessary) and passing that to the BC?

**Reply by GiudGiud** [reply]
like this one https://mooseframework.inl.gov/source/functions/PiecewiseLinear.html
using a `t` axis and data from CSV parameter

**Reply by vabufano** [reply]
Thank you both. I will definitely try to use the solution proposed. Thanks!
